### DISCUSSION TITLE
Nonebot项目首次运行报错，求助

### DISCUSSION THREAD

**Discussion by ETO-J** [nonebot/discussions#525] [问答] [answered]
![image](https://github.com/user-attachments/assets/d431693e-7c00-4a92-a254-eeca8d4a7476)
安装完成环境后，nb create运行正常，适配器是QQ官方机器人，驱动是fastapi。然后运行nb qqbot-test --reload时出现如图的报错。
不知道哪里出了问题？

**Comment by ETO-J** [comment]
系统是windows2022 服务器端

**Comment by yanyongyu** [comment, marked as answer]
你的截图乱七八糟的啥都看不出来啊

**Reply by ETO-J** [reply]
这个就是nb run --reload 运行的结果报错，再往下截图是无关的东西了

**Reply by yanyongyu** [reply]
你这报错啥都没有，显示还是混乱的没法看。QQ适配器要http client和ws client驱动器，你需要添加httpx websockets或者aiohttp驱动器。

**Reply by ETO-J** [reply]
感谢，改了驱动器，已经解决
